src/Modules/UngradedModules.jl

```diff
@@ -1539,8 +1543,18 @@ function is_canonically_isomorphic(M::SubModuleOfFreeModule, N::SubModuleOfFreeM
   return SubModuleOfFreeModule(G, [f(v) for v in gens(M)]) == N
 end
 
-#+(M::SubModuleOfFreeModule, N::SubModuleOfFreeModule) = sum(M, N)
+Base.:+(M::SubModuleOfFreeModule, N::SubModuleOfFreeModule) = sum(M, N)
+
+function Base.:*(I::MPolyIdeal{T}, M::SubModuleOfFreeModule{T}) where {T<: MPolyRingElem}
+  F = ambient_free_module(M)
+  newgens = filter( x -> !is_zero(x),[a*b for a in gens(I) for b in gens(M)])
+  return oscar.SubModuleOfFreeModule(F, newgens)
+end
 
+function times(I::MPolyIdeal, M::FreeMod)
```

Comment:
I would overload the  operator here. @fieker 
```suggestion
`Base.(*)(I::MPolyIdeal, M::FreeMod)`
```

Reply:
This breaks something in the ReesAlgebra-Tests and I have not found out yet, how to repair it. So for the moment I used another name for it to get the true content of the PR running first and investigate later on (but before merge!).

Reply:
There is the following already 
`*(I::Ideal, M::ModuleFP) in Oscar at .julia/dev/Oscar/src/Modules/UngradedModules.jl:6500`
Are your modules graded?

Reply:
The problem is not finding this function (which happens to return a subquo), but my lack of knowledge concerning the structural decisions in the design of the (ungraded) modules. I naively wrote the duVal-Tester using SubModuleOfFreeModule -- because this type tab-completed and seemed to encode precisely what I needed to prepare a submodule of the form N + J*M in a free module M. This may have been a wrong decision on my side -- its origin is the intrinsic analogy of the duVal-singularity tester between the hypersurface&ideal case and the non-hypersurface&module case. I will probably have to rewrite everything in terms of sub and quo.